src/script/index.js

```diff
@@ -102,11 +103,26 @@ const buildInfo = {
 	buildNumber: '__BUILD_NUMBER__' || null
 };
 
+import validateGraphF from './format/graphValidator'; // eslint-disable-line
+
 const ketcher = module.exports = Object.assign({ // eslint-disable-line no-multi-assign
 	getSmiles,
 	saveSmiles,
 	getMolfile,
 	setMolecule,
 	addFragment,
-	showMolfile
+	showMolfile,
+
+	// TODO: remove it
+	toGraph: () => {
+		const j = graph.toGraph(ketcher.editor.render.ctab.molecule);
+		validateGraphF(j);
+		return j;
+	},
+	fromGraph: () => graph.fromGraph(graph.toGraph(ketcher.editor.render.ctab.molecule)),
```

Comment:
Remove `loadStruct` API - was used as debug, replace `toGraph` with `getGraph` and write correct method for `setGraph`